docs/actions.md

```diff
@@ -216,10 +216,17 @@ ok: invoked /whisk.system/samples/greeting with id 5975c24de0114ef2b5c24de0118ef
 A blocking invocation request will _wait_ for the activation result to be available. The wait period
 is the lesser of 60 seconds or the action's configured
 [time limit](reference.md#per-action-timeout-ms-default-60s).
-The result of the activation is returned if it is available within the wait period.
+
+The result of the activation is returned if it is available within the blocking wait period (this wait period is by default 60 seconds as well).
 Otherwise, the activation continues processing in the system and an activation ID is returned
 so that one may check for the result later, as with non-blocking requests
 (see [here](#watching-action-output) for tips on monitoring activations).
+If execution of the action exceeds the action's configured time limit within the blocking wait period
+the result of the invocation is an error caused by the action timeout and the response status is _application error_.
```

Comment:
I don't think we've explained application error or response status at this point; this happens in the next section (understanding the activation record). How about this:

When an action exceeds its configured time lime, the activation record will indicate this error. See [understanding the activation record] for more details.

or something to that effect.

Reply:
Could be.